packages/experimental/Shimmer/src/Shimmer.styling.ts

```diff
@@ -0,0 +1,52 @@
+import { UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { shimmerName, ShimmerProps, ShimmerSlotProps, ShimmerTokens } from './Shimmer.types';
+export const stylingSettings: UseStylingOptions<ShimmerProps, ShimmerSlotProps, ShimmerTokens> = {
+  tokens: [
+    (t) => ({
+      toValue: 30,
+      duration: 7000,
+      delay: 0,
+      gradientTintColor: t.host?.appearance === 'light' ? 'white' : 'black',
+      shimmerTintColor: t.host?.appearance === 'light' ? "#E1E1E1" : '#404040',
+      width: 200,
+      height: 100,
+      angle: 0,
+      gradientOpacity: .7,
+    }),
+    shimmerName,
+  ],
+  slotProps: {
+    root: buildProps(
+      (tokens: ShimmerTokens) => ({
+        gradientTintColor: tokens.gradientTintColor,
+        toValue: tokens.toValue,
+        duration: tokens.duration,
+        delay: tokens.delay,
+        shimmerTintColor: tokens.shimmerTintColor,
+        width: tokens.width,
+        height: tokens.height,
+        angle: tokens.angle,
+        gradientOpacity: tokens.gradientOpacity,
+        accessibilityRole: 'progressbar',
+        accessibilityLabel: "Loading content",
```

Comment:
This isn't localized, and I have a feeling that could cause problems down the line. What happens if we just set the role and not the label? And if this is required, I think downstream in sdx_platform there is localization code set up so we could defer responsibility to clients to set the label until we have a better story here.

Reply:
Good catch. If I don't specify the label, VO would just read out the accessibility role label. The prop isn't required here so I will remove it.